.github/workflows/tox.yml

```diff
@@ -85,3 +85,46 @@ jobs:
           if [[ "$TOXENV" == 'csslint' ]]; then npm install -g csslint; fi
       - name: Run tox
         run: python -m tox
+
+  translation:
+    strategy:
+      fail-fast: false
+      max-parallel: 4
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        python-version: [3.6, 3.7, 3.8, 3.9, pypy3]
+        include:
+          - python-version: 3.6
+            tox-env: py36
+          - python-version: 3.7
+            tox-env: py37
+          - python-version: 3.8
+            tox-env: py38
+          - python-version: 3.9
+            tox-env: py39
+          - python-version: pypy3
+            tox-env: pypy3
```

Comment:
Is it really important to check that there is no diff on Python 3.6 on Linux *and* no diff on Python 3.9 on Windows *and* *and* *and*...
Just seems wasteful.
And, well, if there ever is actually a difference across different versions, then this check would be impossible to satisfy.